test/address_sanitizer_run_eunit.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+
+echo "============================================"
+echo "Note that the \"test/address_sanitizer_setup.sh\" has to run"
+echo "successfully at least once before this script will work."
+echo "============================================"
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+OLD_DIR=`pwd`
+
+cd "$SCRIPT_DIR/../otp"
+
+
+export ERL_TOP=`pwd`
+export PATH=$ERL_TOP/bin:$PATH
+    
+echo "Changed ERL_TOP ($ERL_TOP) and PATH"
+
+cerl -asan -eval "erlang:halt()"
+
+cd "$SCRIPT_DIR/.."
+
+(JQ_MEMSAN_DEBUG=1 rebar3 eunit ; true)
```

Comment:
Thanks! This is a left over from experimentation when I had turned on stop on command failure and I think it can be simplified to:

Q_MEMSAN_DEBUG=1 rebar3 eunit